elements/rh-secondary-nav/demo/analytics.html

```diff
@@ -0,0 +1,209 @@
+<link rel="stylesheet" href="demo.css">
+<link rel="stylesheet" href="../rh-secondary-nav-lightdom.css">
+<script type="module" src="rh-secondary-nav.js"></script>
+
+<div class="mock-nav"></div>
+<rh-secondary-nav role="navigation" data-analytics-region="Ansible secondary nav">
```

Comment:
@jvanpelt leaving this comment here to block auto-merge on your approval. Please resolve this comment (and any other that come up for your review) to enable auto merge

Reply:
Thanks @bennypowers - I committed the suggestions you added, and the preview appears correctly now.

Reply:
Can you make the region something like `data-analytics-region="secondary-navigation"`.  Do not want it to be site specific since we will use same tracking across all.